Source/Scene/WebMapServiceImageryProvider.js

```diff
@@ -239,6 +239,37 @@ function WebMapServiceImageryProvider(options) {
         ? "EPSG:3857"
         : "CRS:84"
     );
+
+    // The axis order in previous versions of the WMS specifications was to always use easting (x or lon ) and northing (y or
+    // lat). WMS 1.3.0 specifies that, depending on the particular CRS, the x axis may or may not be oriented West-to-East,
+    // and the y axis may or may not be oriented South-to-North. The WMS portrayal operation shall account for axis order.
+    // This affects some of the EPSG codes that were commonly used such as ESPG:4326. The current implementation
+    // makes sure that coordinates passed to the server (as part of the GetMap BBOX parameter) as well as those advertised
+    // in the capabilities document reflect the inverse axe orders for EPSG codes between 4000 and 5000.
+    //  - Taken from Section 9.1.3 of https://download.osgeo.org/mapserver/docs/MapServer-56.pdf
+    var parts = parameters.crs.split(":");
```

Comment:
@tantricllama Is it all possible for an incorrectly defined imagery provider (with `crs` parameter that isn't a sane EPSG code) to make it to here?

If so then it's possible that the `parts[0]` and `parts[1]` to be `undefined` however in my isolated testing I think that's fine.

If `parts[0]` is `undefined` then we don't end up inside the if statement.

If the `crs` parameter was just "EPSG" then `parts[1]` would be `undefined` and we would make it into the if statement, however `Number(undefined)` returns `NaN` which shouldn't break anything.

So I guess it's okay?